sdk/eventhub/azure-eventhub/stress/dev_requirement.txt

```diff
@@ -2,6 +2,7 @@ psutil
 ..
 ../../azure-eventhub-checkpointstoreblob
 ../../azure-eventhub-checkpointstoreblob-aio
-azure-servicebus
+azure-servicebus==0.50.3
```

Comment:
Is this just for testing purposes right now? But will be removed after merging?

Reply:
actually this is on purpose to stick to sb 0.50.3 at least for current stress test implementation.
the sb v0.50.3 provides the ability to easily create new eventhub resources.

we might want to drop it off and move to using mgmt lib at some point, but that would be a separate issue&PR.